apps/docs/pages/guides/realtime/quotas.mdx

```diff
@@ -0,0 +1,112 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'quotas',
+  title: 'Realtime Quotas',
+  description: 'Understanding Realtime quotas',
+  sidebar_label: 'Quotas',
+}
+
+<Admonition type="note">
+  Upgrade your plan to increase your quotas. Without a spend cap, or on an Enterprise plan, some
+  quotas are still in place to protect budgets. All quotas are configurable per project. [Contact
+  support](https://app.supabase.com/support/new) if you need your quotas increased. Our cluster
+  supports millions of concurrent connections and message throughput for production workloads.
+</Admonition>
+
+## Quotas by Plan
+
+You can increase your quotas by upgrading your plan and removing your spend cap. We still have a starting quota set even without a spend cap to protect your budget just in case.
+
+If you need your quotas increased further please [contact support](https://app.supabase.com/support/new).
+
```

Comment:
consider representing as a table

|                          | Free | Pro | PAYG | Team | Enterprise |
|--------------------------|------|-----|------|------|------------|
| Concurrent clients       | 200  |     |      |      |            |
| Messages per second      |      |     |      |      |            |
| Channel joins per second |      |     |      |      |            |

Reply:
@kiwicopple good call.

@chasers i'm currently on docs and will update it to table.

Reply:
@chasers for your review: https://github.com/supabase/supabase/pull/14946